job-dsls/jobs/seed_job.groovy

```diff
@@ -59,7 +59,6 @@ job("a-seed-job") {
                     "job-dsls/jobs/**/new_downstream_production.groovy\n" +
                     "job-dsls/jobs/**/new_upstream.groovy\n" +
                     "job-dsls/jobs/**/deploy_jobs.groovy\n" +
-                    "job-dsls/jobs/**/kie_docs_pr.groovy\n" +
```

Comment:
why is kie-docs PR job removed?

Reply:
because this is now created by the new_pr_jobs.groovy